dags/probe_scraper.py

```diff
@@ -81,7 +81,6 @@
         name='schema-generator-1',
         namespace='default',
         image='mozilla/mozilla-schema-generator:latest',
-        image_pull_policy='Always',
```

Comment:
Just a suggestion that it may be helpful for this GkePodOperator to keep this param so that others writing new dags can reference it.

Reply:
Hmm, can you clarify? The `GkePodOperator` does have this param now

Reply:
Just if anyone wanted to modify image_pull_policy, and were to git grep for image_pull_policy, they wouldn't find anything. But that's prob okay if they reference the upstream code for whatever deploy tag we're on